styles/styles.css

```diff
@@ -160,15 +162,6 @@ header {
 
 
 @media (min-width: 990px) { /* desktop */
-  :root {
-    --heading-font-size-xxl: 60px;
```

Comment:
@andreibogdan Do you know where a header this large was used ?

Reply:
I did find any trace of that font to be used. Maximum heading font size is 56px for large desktops view.